common/assets/scss/overrides/_sticky-sidebar.scss

```diff
@@ -0,0 +1,7 @@
+.app-\!-not-sticky-mobile {
```

Comment:
Do we need a separate override class? Feels like it could be part of the `sticky-sidebar` module as can't see any scenarios where we want it to display how it is on mobile.

Reply:
good point 👍